apps/components/src/app/toolbar/app.ts

```diff
@@ -0,0 +1,141 @@
+import { Component } from '@angular/core';
+import { NgIcon, provideIcons } from '@ng-icons/core';
+import {
+  heroBars3,
+  heroBars3BottomLeft,
+  heroBars3BottomRight,
+  heroCog6Tooth,
+  heroDocument,
+  heroFolder,
+} from '@ng-icons/heroicons/outline';
+import { NgpButton } from 'ng-primitives/button';
+import { NgpRovingFocusGroup, NgpRovingFocusItem } from 'ng-primitives/roving-focus';
+import { NgpSeparator } from 'ng-primitives/separator';
+import { NgpToggleGroup, NgpToggleGroupItem } from 'ng-primitives/toggle-group';
+import { Toolbar } from './toolbar';
+
+@Component({
+  selector: 'app-toolbar-example',
+  imports: [
+    Toolbar,
+    NgIcon,
+    NgpButton,
+    NgpRovingFocusGroup,
+    NgpRovingFocusItem,
+    NgpSeparator,
+    NgpToggleGroup,
+    NgpToggleGroupItem,
+  ],
+  providers: [
+    provideIcons({
+      heroDocument,
+      heroFolder,
+      heroBars3BottomLeft,
+      heroBars3,
+      heroBars3BottomRight,
+      heroCog6Tooth,
+    }),
+  ],
+  template: `
+    <div app-toolbar>
+      <button type="button" ngpButton ngpRovingFocusItem>
+        <ng-icon name="heroDocument" />
+      </button>
+      <button type="button" ngpButton ngpRovingFocusItem>
+        <ng-icon name="heroFolder" />
+      </button>
```

Comment:
I wonder might it be good to also add a toolbar button component here that adds `NgpButton` and `NgpRovingFocusItem` as host directives and also sets type to button. Might be convenient for people.

Then for the example if we just show a few toolbar buttons in the toolbar that would suffice, we can let users add more complex things like toggle groups only if they want.